### ISSUE TITLE
Rewording

### ISSUE THREAD

**Issue by ErinBecker** [datacarpentry/R-ecology-lesson#212]
In this page: http://www.datacarpentry.org/R-ecology-lesson/01-intro-to-R.html:

a. Under "Creating Objects", it's a bit clearer to say "...we may want to convert this weight *to* pounds..." rather than "...in pounds..." as currently written.

b. Under "Functions and their arguments", I suggest rewording "Functions are "canned scripts" that automate something complicated or convenient or both." to "Functions are 'canned scripts' that automate more complicated sets of commands, including operations, assignments, etc."

c. Under "Call Out", I would suggest changing the subject/verb agreement from "What is called objects in R is known as variables in many other programming languages" to "What *are* called objects in R *are* known as variables in many other programming languages"

-- submitted by email to checkout@carpentries.org

**Comment by fmichonneau** [comment]
I agree with the proposed changes
